M2/BUILD/mahrud/Makefile

```diff
@@ -0,0 +1,24 @@
+# 
+# Written for Fedora 26
+#
+
+.PHONY: always
+
+rpm-docker-build:
+	[ ! -z `docker images -q m2-rpm` ] || docker build -t m2-rpm .
+	docker run -it \
+	  -v `pwd`/../../..:/M2 \
+	  m2-rpm \
+	  make rpm-build -j4 -C /M2/M2/BUILD/mahrud
+
+rpm-build:
+	mkdir -p build;
+	make -j4 -C ../..;
+	cd build && ../../../configure \
+	  --enable-rpm \
+	  --prefix=/usr \
+	  --enable-download \
+	  --enable-build-libraries="factory pari flint ntl fflas_ffpack memtailor mathic mathicgb";
```

Comment:
I might have fixed that on the master branch -- it seems familiar.  Could you try the master branch?